README.md

```diff
@@ -61,7 +61,7 @@ The list strives to be complete rather than [awesome](https://github.com/sindres
 * [WDOM](https://github.com/miyakogi/wdom).  "WDOM is a Python GUI library for browser-based desktop applications."
 
 ### Ruby
-
+* [Turbo](https://turbo.hotwired.dev/). "The speed of a single-page web application without having to write any JavaScript."
```

Comment:
Turbo isn't specific to Ruby - it's pure JS that work with any backend

Reply:
should we add language agnostic section? what is your suggestion?